src/mscorlib/src/System/String.Manipulation.cs

```diff
@@ -1270,7 +1272,7 @@ private String[] SplitInternal(String separator, String[] separators, Int32 coun
         //     the original string will be returned regardless of the count. 
         //
 
-        private String[] SplitKeepEmptyEntries(Int32[] sepList, Int32[] lengthList, Int32 defaultLength, Int32 numReplaces, int count)
+        private string[] SplitKeepEmptyEntries(Span<int> sepList, Span<int> lengthList, int defaultLength, int numReplaces, int count)
```

Comment:
Change `sepList` and `lengthList` to be typed as `ReadOnlySpan<int>` instead of `Span<int>` since the spans are not modified in this method? The call sites wouldn't need to change as there's an implicit conversion from `Span<T>` to `ReadOnlySpan<T>`.

Reply:
done, thanks